src/services/user.js

```diff
@@ -316,42 +316,24 @@ const getMemberships = async id => {
   return memberships;
 };
 
```

Comment:
Do we have a convention for comment format?  In particular, do we prefer 
  // Comment
or
  //Comment
?
(Personally, I prefer a space after //.  But being consistent with existing code is more important than personal preference.)